server/errorHandler.ts

```diff
@@ -4,7 +4,7 @@ import logger from '../logger'
 
 export default function createErrorHandler(production: boolean) {
   return (error: HTTPError, req: Request, res: Response, next: NextFunction): void => {
-    logger.error(`Error handling request for '${req.originalUrl}', user '${res.locals.user?.username}'`, error)
+    logger.error(`Error handling request for '${req.originalUrl}', user '${req.user.sub.slice(0, 30)}...'`, error)
```

Comment:
`sub` is quite a long string so just showing a portion in error log to help with debugging.